Testing/SystemTests/tests/analysis/ILLDirectGeometryReductionTest.py

```diff
@@ -145,7 +145,7 @@ def runTest(self):
         mtd['cropped'].mutableRun().addProperty('run_title', '', True)
 
     def validate(self):
-        self.tolerance = 1e-7
+        self.tolerance = 1e-3
```

Comment:
This is a dramatic change in tolerance. Maybe updating the reference and keeping the tolerance is a better idea. Can you post a comparison of the output of this test with the current reference?